ambry-account/src/main/resources/AccountSchema.ddl

```diff
@@ -18,11 +18,12 @@ CREATE TABLE IF NOT EXISTS Accounts
     version INT NOT NULL,
     creationTime DATETIME(3) NOT NULL,
     lastModifiedTime DATETIME(3) NOT NULL,
-    accountId INT GENERATED ALWAYS AS (accountInfo->>"$.accountId") STORED NOT NULL,
-    accountName VARCHAR(255) GENERATED ALWAYS AS (accountInfo->>"$.accountName") NOT NULL,
-    status VARCHAR(50) GENERATED ALWAYS AS (accountInfo->>"$.status") STORED NOT NULL,
+    accountId INT NOT NULL,
+    accountName VARCHAR(255) NOT NULL,
+    status VARCHAR(50) NOT NULL,
+    deleted_ts datetime(6) DEFAULT NULL,
```

Comment:
Just a question. Is this a new field? Looks like it is not being used in sql queries in `AccountDao.java`.

Reply:
Yes, this field is recommend by mysql team while we are moving from native replication to gg. 
And it can also be used in the future when we want to delete the entire record.